ci-operator/config/medik8s/node-maintenance-operator/medik8s-node-maintenance-operator-main.yaml

```diff
@@ -0,0 +1,54 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: ocp
+    tag: base
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```

Comment:
Not sure what the impact of go version we are using here.
Is there any risk when it's different than the one we use in the operator ?

Reply:
+1, always use the same Go version everywhere, I remeber having an issue wherre even a new patch version introduced an issue (root cause was bad code, but anyway...)

Reply:
Ok :)